Chapter3_Functions/Chapter 3 #2.js

```diff
@@ -0,0 +1,13 @@
+function isEven(n)
+{
+    if(n==0)
+    return true;
+    else if(n==1)
+    return false;
+    else if(n<0)
+    console.log("WRONG INPUT");
```

Comment:
Why "Wrong Input"? Negative numbers can also be differentiated as even and odd and they should definitely be handled.